packages/apps-config/src/endpoints/testing.ts

```diff
@@ -291,6 +291,13 @@ export const testChains: EndpointOption[] = [
       'KILT Protocol': 'wss://peregrine.kilt.io/parachain-public-ws/'
     }
   },
+  {
+    info: 'kintsugi-testnet',
+    text: 'Kintsugi Testnet',
+    providers: {
+      Interlay: 'wss://api-dev-kintsugi.interlay.io/parachain',
```

Comment:
```suggestion
      Interlay: 'wss://api-dev-kintsugi.interlay.io/parachain'
```

I believe this is what CI is complaining about.